ConformanceExamples/README.md

```diff
@@ -0,0 +1,7 @@
+# SwiftRef:  Protocol Conformance Examples
```

Comment:
Column limit please.

Reply:
I know appreciate the benefits of column width.  Thank you.